docs/contributing/building/macosx.md

```diff
@@ -33,8 +33,11 @@ Tracee isn't meant to run in OSX but, with Docker, you're able to test building
 it. If you try to run it from the building environment you may get the following
```

Comment:
"you're able to test building it"

Change this to "you can test building it".

is "building environment" a concept? Maybe "build environment" is better?